### PR TITLE
Refs #29993 - Remove before_action puppetrun

### PR DESCRIPTION
**Pull request by domitea**
Remove missing oneliner for puppetrun removal

### CODE REVIEW

**Review by ezr-ondrej — DISMISSED** [review]
Thanks @domitea ! :+1:
WWFJ

**Comment by ezr-ondrej** [comment]
Could you update the commit message to `Refs #xxxx` -

**Comment by domitea** [comment]
@ezr-ondrej looks better now?

**Review by ezr-ondrej — DISMISSED** [review]
:+1:

File: app/controllers/concerns/foreman/controller/puppet/hosts_controller_extensions.rb [hunk 1]
```diff
@@ -14,7 +14,6 @@ module Foreman::Controller::Puppet::HostsControllerExtensions
     alias_method :ajax_request_for_puppet_host_extensions, :ajax_request
 
     before_action :ajax_request_for_puppet_host_extensions, :only => PUPPET_AJAX_REQUESTS
-    before_action :find_resource_for_puppet_host_extensions, :only => [:puppetrun]
```
Comment (tbrisker) [inline app/controllers/concerns/foreman/controller/puppet/hosts_controller_extensions.rb:17]: Please remove the alias for it on line 13 as well

**Comment by domitea** [comment]
@tbrisker @ezr-ondrej Alias removed

**Review by ezr-ondrej — APPROVED** [review]

**Comment by tbrisker** [comment]
Thanks @domitea and @ezr-ondrej !

**Pull request merged**